maltoolbox/attackgraph/attackgraph.py

```diff
@@ -195,6 +195,7 @@ def _to_dict(self):
 
     def save_to_file(self, filename):
         """Save to json/yml depending on extension"""
+        logger.debug(f'Save attack graph to file\"{filename}\".')
```

Comment:
Might be worth to use the new format which is used in #49:
```
logger.debug('Save attack graph to file "%s".', filename) 
```

(I don't know how to make 'suggestions' in github)